docs/extensibility/170-additional-sections-wizard.md

```diff
@@ -0,0 +1,29 @@
+# Additional Sections for Wizard Extensions
+
+## _steps_ section
+
+Each wizard consists of steps. This section contains their definitions.
+
+### Available Parameters
+
+| Parameter       | Required | Type   | Description                                                                                                                    |  |
+|-----------------|----------|--------|--------------------------------------------------------------------------------------------------------------------------------|--|
+| **name**        | **Yes**  | string | The step name displayed on the step navigation and in the step header.                                                         |  |
+| **resource**    | **Yes**  | string | The default resource identifier for this step.                                                                                 |  |
+| **form**        | **Yes**  | string | the form definition. This is analogous to the contents of the [_form_ section](./40-form-fields.md) of the resource extension. |  |
+| **description** | No       | string | Additional details about the step, shown only when the step is active.                                                         |  |
+
```

Comment:
Can we add a short example here?

Reply:
We have examples of the wizards here: https://github.com/kyma-project/busola/tree/main/examples/wizard you can reuse things from this file